client/src/game/Game.js

```diff
@@ -55,14 +62,20 @@ const Game = (props) => {
           player.offsetY = 0;
           player.top = floor - player.height;
           player.jumping = false;
+          if (player.direction === "JUMPING") {
+            player.direction = "FRONT";
+          }
```

Comment:
Similar to previous comment.

```suggestion
          if (player.direction === DIRECTIONS.JUMPING) {
            player.direction = DIRECTIONS.FRONT;
          }
```